pyproject.toml

```
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "py-assetman"
+description = "assetman assetmanager"
+version = "0.3.0rc"
+authors = [
+  { name="Will McCutchen", email="wm@bit.ly" },
+]
+maintainers = [
+  { name="Jehiah Czebotar", email="jehiah@gmail.com"},
+  { name="Josh Harshman", email="josh.harshman@bit.ly"}
+]
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache 2.0",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+  "boto3>=1.34.15",
+  "tornado>=6.3.3",
```

Since we don't have any specific constraints with these packages - let's just make this broad.

```suggestion
  "boto3>=1.10.0",
  "tornado>=5.1.1",
```